script/outside.js

```diff
@@ -364,11 +364,11 @@ var Outside = {
 	
 	updateVillageRow: function(name, num, village) {
 		var id = 'building_row_' + name.replace(' ', '-');
-		name = _(name);
+		lname = _(name);
```

Comment:
Needs `var` or you're declaring it in the global scope. Polluting the global scope is bad.